lib/elixir/lib/kernel/utils.ex

```diff
@@ -19,7 +22,8 @@ defmodule Kernel.Utils do
   @doc """
   Callback for defdelegate.
   """
-  def defdelegate(fun, opts) do
+  @spec defdelegate(Macro.t, Keyword.t) :: [tuple]
```

Comment:
No need to spec this function. This is a callback for internal implementation details.